modules/SQLiteTreeView.jsm

```diff
@@ -91,9 +89,7 @@ class SQLiteTreeView {
 	 * @return {void}
 	 */
 	_updateOrderClause() {
-		this.orderClause = this.sortOrder.map(function (elem) {
-			return this.columns[elem.index] + (elem.orderDesc ? " DESC": "");
-		}, this).join(", ");
+		this.orderClause = this.sortOrder.map(e => this.columns[e.index] + (e.orderDesc ? " DESC": ""), this).join(", ");
```

Comment:
Note that one difference between "normal" functions and arrow functions is how `this` behaves.

If you are not familiar with that, see e.g. the following explanation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this#arrow_functions
There are probably also a lot of other resources who go over the difference in more detail.

How is this relevant here?
From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#iterative_methods:
> The `thisArg` argument is irrelevant for any `callbackFn` defined with an [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), as arrow functions don't have their own `this` [binding](https://developer.mozilla.org/en-US/docs/Glossary/Binding).

So you should be able to drop the `this` argument to `map`. But best test it.

Reply:
Hi @lieser ,
thank you again for the feedback. Since my sole purpose was to improve the readability for the future, I simply restored the old function definition in this case.

And I didn't know about that fact on this.